Filtration/Resources/Enchantments.txt

```diff
@@ -0,0 +1 @@
+﻿Enchantment Decree of Force
```

Comment:
Shouldn't there be more than one enchantment in this file? There's hundreds listed here: https://pathofexile.gamepedia.com/Enchantment 

Not sure if there's a better source of the data